isis/src/docsys/build/redirect.xml

```diff
@@ -16,8 +16,8 @@
   </table>
   </div>
 
-  <meta http-equiv="refresh" content="0; url='7.0.0/index.html'" />
+  <meta http-equiv="refresh" content="0; url='8.1.0/'" />
```

Comment:
I imagine we would need to update the release instructions on https://github.com/DOI-USGS/asc-public-docs to make sure this is updated regularly?

Reply:
Removed this redirect.xml file in replacement of a lambda redirect.